src/fonts/common_metrics.ts

```diff
@@ -1,75 +1,6 @@
 export const CommonMetrics = {
   smufl: true,
 
-  ornament: {
-    brassScoop: {
-      xOffset: -12,
-      yOffset: 0,
-      stemUpYOffset: 0,
-      reportedWidth: 20,
-    },
-    brassDoitMedium: {
-      xOffset: 16,
-      yOffset: 0,
-      stemUpYOffset: 0,
-      reportedWidth: 22,
-    },
-    brassFallLipShort: {
-      xOffset: 16,
-      yOffset: 0,
-      stemUpYOffset: 0,
-      reportedWidth: 15,
-    },
-    brassLiftMedium: {
-      xOffset: 16,
-      yOffset: 5,
-      stemUpYOffset: 0,
-      reportedWidth: 5,
-    },
-    brassFallRoughMedium: {
-      xOffset: 16,
-      yOffset: 28,
-      stemUpYOffset: 0,
-      reportedWidth: 5,
-    },
-    brassBend: {
-      xOffset: 2,
-      yOffset: -8,
-      stemUpYOffset: 25,
-      reportedWidth: 5,
-    },
-    brassMuteClosed: {
-      xOffset: 3,
-      yOffset: -8,
-      stemUpYOffset: 25,
-      reportedWidth: 5,
-    },
-    brassMuteOpen: {
-      xOffset: 3,
-      yOffset: -7,
-      stemUpYOffset: 25,
-      reportedWidth: 5,
-    },
-    brassFlip: {
-      xOffset: 10,
-      yOffset: 0,
-      stemUpYOffset: 7,
-      reportedWidth: 10,
-    },
-    brassJazzTurn: {
-      xOffset: 0,
-      yOffset: 0,
-      stemUpYOffset: 8,
-      reportedWidth: 31,
-    },
-    brassSmear: {
-      xOffset: 10,
-      yOffset: 0,
-      stemUpYOffset: 8,
-      reportedWidth: 5,
-    },
-  },
-
```

Comment:
I assume these were all hand-tuned at some point. If we need to improve the visual layout of the ornaments, we should figure out how to possibly reincorporate this visual hand tuning.